### ISSUE TITLE
Saml2AuthenticationRequest missing several properties

### ISSUE THREAD

**Issue by lukos** [KentorIT/authservices#208]
The class `Kentor.AuthServices.Saml2P.Saml2AuthenticationRequest` is missing several of the properties from the spec, such as IsPassive, Subject and Scoping. I assume these are not missing for a reason, they just weren't added.

Can these be added in? I am happy to make the changes in a Pull Request if this is correct.

**Comment by AndersAbel** [comment]
You're right in that they were just not added. A pull request with that would be welcome.

If you haven't allready seen it I suggest that you read the [contribution guidelines](https://github.com/KentorIT/authservices/blob/master/doc/Contributing.md) before coding.

**Comment by jimmytoenners** [comment]
Scoping is a needed feature. Did anyone work on this yet? 
I my fork I have implemented Saml2Scoping as a property on the IdentityProvider and Saml2AuthenticationRequest. When a Saml2AuthenticationRequest is created it simply takes the scoping from the idp and applies it. We will soon need it on a per request base and will look into this
